x-pack/plugins/security_solution/public/timelines/components/timeline/eql_tab_content/index.tsx

```diff
@@ -57,12 +58,12 @@ import { Sourcerer } from '../../../../common/components/sourcerer';
 import { useLicense } from '../../../../common/hooks/use_license';
 import { HeaderActions } from '../../../../common/components/header_actions/header_actions';
 
-const TimelineHeaderContainer = styled.div`
-  margin-top: 6px;
+const EqlTabHeaderContainer = styled.div`
+  margin-top: ${euiThemeVars.euiSizeS};
   width: 100%;
 `;
 
-TimelineHeaderContainer.displayName = 'TimelineHeaderContainer';
+EqlTabHeaderContainer.displayName = 'TimelineHeaderContainer';
```

Comment:
should we change the displayName at the same time from `TimelineHeaderContainer` to `EqlTabHeaderContainer`?

Reply:
fixed in af2d7372da4e3a90820781cbc60322d48e224002